build.gradle

```diff
@@ -620,6 +620,11 @@ project('test:system') {
     // Specifically publish this test project
     apply plugin: 'maven'
 
+    // override the compile java options to disable -Werror. This will be removed once https://github.com/kubernetes-client/java/issues/436
+    compileJava {
+        options.compilerArgs.remove("-Werror")
```

Comment:
Would adding `-Xlint:-processing` work and at least scope down the damage? It is overkill to disable compilation failure on warnings altogether, even if it just for system tests.

Reply:
All the -Xlint options included in java.gradle (https://github.com/pravega/pravega/blob/master/gradle/java.gradle#L30) still apply and all Warnings will be displayed. Only the `-Werror` is disabled for test:system project.